apps/web/src/app/(networks)/(non-evm)/stellar/_common/ui/Swap/simple/simple-swap-execute-button.tsx

```diff
@@ -198,10 +195,9 @@ export const SimpleSwapExecuteButton = () => {
   } = useNeedsTrustlines(intermediateTokens)
 
   // Check if any trustlines are needed (block swap until created)
+  // Note: Only check output token and intermediate tokens - user already has input token
   const needsAnyTrustline =
-    needsToken0Trustline ||
-    needsToken1Trustline ||
-    needsAnyIntermediateTrustline
+    needsToken1Trustline || needsAnyIntermediateTrustline
```

Comment:
Non-blocking nit: we probably don't need the `needsAnyIntermediateTrustline` check https://hypotenuse.slack.com/archives/C08D14YQKNV/p1765993057350599?thread_ts=1765977730.423999&cid=C08D14YQKNV

Reply:
We can probably also remove the conditional rendering of trustline warning components for the intermediate tokens.